llarp/router/router.hpp

```diff
@@ -242,15 +255,16 @@ namespace llarp
     NetworkConfig networkConfig;
     DnsConfig dnsConfig;
 
-    const IpAddress DefaultRPCBindAddr = IpAddress("127.0.0.1:1190");
+    const std::string DefaultRPCBindAddr = "tcp://127.0.0.1:1190";
```

Comment:
You could perhaps switch this to a `lokimq::address`.

Reply:
Can also be written as:
```C++
const auto DefaultRPCBindAddr = lokimq::address::tcp("127.0.0.1", 1190);
```

Reply:
i want to support unix sockets too

Reply:
Right, but this is just for the default value -- a `lokimq::address` can hold either tcp/ipc or the +curve variants.